x/stakeibc/keeper/msg_server_clear_balance.go

```diff
@@ -0,0 +1,62 @@
+package keeper
+
+import (
+	"context"
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	ibctransfertypes "github.com/cosmos/ibc-go/v3/modules/apps/transfer/types"
+	"github.com/spf13/cast"
+
+	"github.com/Stride-Labs/stride/x/stakeibc/types"
+)
+
+func (k msgServer) ClearBalance(goCtx context.Context, msg *types.MsgClearBalance) (*types.MsgClearBalanceResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	zone, found := k.GetHostZone(ctx, msg.ChainId)
+	if !found {
+		return nil, sdkerrors.Wrapf(types.ErrInvalidHostZone, "chainId: %s", msg.ChainId)
+	}
+	feeAccount := zone.GetFeeAccount()
+	if feeAccount == nil {
+		return nil, sdkerrors.Wrapf(types.ErrFeeAccountNotRegistered, "chainId: %s", msg.ChainId)
+	}
+
+	// should this be a param? the transfer port _should_ always be the same across zones
+	sourcePort := "transfer"
```

Comment:
can we get this dynamically? thinking about sdk upgrades

Reply:
good call! done